integration-tests/tests/multichain/actions/mod.rs

```diff
@@ -58,6 +59,45 @@ pub async fn request_sign(
     Ok((payload, account, tx_hash))
 }
 
+pub async fn request_sign_sync(
```

Comment:
we shouldn't need to test both versions though. They're pretty much the same, like there's no difference besides the calling mechanism. They're going to be hitting the same code path in near node side anyways

Reply:
We'll fix this one later

Reply:
I understand that we are testing the same code from our perspective. It's more of a "let's check if MPC returns the signature in time" test. Can be done differently, but this one seems the ost convenient for me.